### PR TITLE
Adding ads.18f.gov to govcloud DNS

### PR DESCRIPTION
**Pull request by wongperson**
Guinea pig #1

Note: PRs affecting cloud.gov or Federalist must receive approval from a member of the relevant team.

### CODE REVIEW

File: terraform/18f.gov.tf [hunk 1]
```diff
@@ -107,6 +107,17 @@ resource "aws_route53_record" "18f_gov_qegrzvzekq4wiompgqufe4xwmarm37lh__domaink
   records = ["qegrzvzekq4wiompgqufe4xwmarm37lh.dkim.amazonses.com"]
 }
 
+resource "aws_route53_record" "18f_gov_ads_18f_gov_a" {
+  zone_id = "${aws_route53_zone.18f_gov_zone.zone_id}"
+  name = "ads.18f.gov."
+  type = "A"
+  alias {
+    name = "<d1p50apr0w92d2.cloudfront.net>."
```
Comment (wslack) [inline terraform/18f.gov.tf]: no brackets on each side

**Comment by wslack** [comment]
Travis is unhappy with line 111.

**Comment by wslack** [comment]
@datn any idea what's breaking here?

**Comment by wslack** [comment]
I've checked and confirmed that line 111 has the same content as other lines in Terraform file.

**Comment by wslack** [comment]
@wongperson how did you do that paste? I the google doc has spaces only

**Comment by wongperson** [comment]
I plopped it into a local notepad as I was taking notes.  Maybe spaces translated into tab characters in there?

**Comment by wslack** [comment]
@wongperson somehow you dropped in some spaces that were `00A0` in unicode instead of `0020`

**Comment by wongperson** [comment]
🤷‍♂️  no idea

**Pull request merged**